lev-api.md

```diff
@@ -478,6 +481,10 @@ Get a list of births. Returns an empty list on no results found
     + id: 500152046 (required)
 
 + Request
+
+    + Parameters
+    
+        + id: 500152046 (required)
```

Comment:
I think I wasn't paying attention to what I'd been messing with before :s Will remove